recipes/libfork/all/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  # Newer versions at the top
+  "2.1.1":
```

Comment:
Hey @uilianries thanks a lot for your help. Indeed, this version was published since I last touched index. I will get back to the author so he can upload the new version.